Verifying sv/ldv-linux-3.0/usb_urb-drivers-media-dvb-ttusb-dec-ttusb_dec.ko_false-unreach-call.cil.out.i.pp.i
main using GenerateFunctionCallPreprocessor ...
GenerateFunctionCallPreprocessor parents for __VERIFIER_error:
__VERIFIER_error
  ldv_blast_assert
    usb_free_urb
      ttusb_dec_exit_rc
        ttusb_dec_disconnect
          main
          <end>
      ttusb_dec_free_iso_urbs
        ttusb_dec_alloc_iso_urbs
          ttusb_dec_init_usb
            ttusb_dec_probe
              main
              <end>
        ttusb_dec_exit_usb
          ttusb_dec_disconnect
            main
            <end>
          ttusb_dec_probe
            main
            <end>
      ttusb_dec_init_usb
        ttusb_dec_probe
          main
          <end>
    usb_free_coherent
      ttusb_dec_exit_rc
        ttusb_dec_disconnect
          main
          <end>
    ldv_check_final_state
      main
      <end>
    ldv_check_final_state
      main
      <end>
